api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/impl/RDFServiceImpl.java

```diff
@@ -317,47 +348,45 @@ protected Model[] separateStatementsWithBlankNodes(Model gm) {
     }
 
     protected Query createQuery(String queryString) throws RDFServiceException {
-        List<Syntax> syntaxes = Arrays.asList(
-                Syntax.defaultQuerySyntax, Syntax.syntaxSPARQL_11,
-                Syntax.syntaxSPARQL_10, Syntax.syntaxSPARQL, Syntax.syntaxARQ);
+        List<Syntax> syntaxes = Arrays.asList(Syntax.defaultQuerySyntax, Syntax.syntaxSPARQL_11, Syntax.syntaxSPARQL_10,
+                Syntax.syntaxSPARQL, Syntax.syntaxARQ);
         Query q = null;
         Iterator<Syntax> syntaxIt = syntaxes.iterator();
         while (q == null) {
```

Comment:
!Objects.nonNull(q) will look better I think, also, I would rename "q" to "query".